tests/test_util/aiohttp.py

```diff
@@ -62,11 +64,15 @@ def request(
         if params:
             url = url.with_query(params)
 
-        self._mocks.append(
-            AiohttpClientMockResponse(
-                method, url, status, content, cookies, exc, headers
-            )
+        new_response = AiohttpClientMockResponse(
+            method, url, status, content, cookies, exc, headers, long_poll
         )
+        if long_poll and content != b"":
```

Comment:
do we need the `!= b""`?
```python
if long_poll and content:
```

Reply:
sure. will change

Reply:
done. i actually moved it earlier in the function, since an empty content is a valid response (i think), but wanted to verify that json, text, and content are all None